sds_data_manager/lambda_code/SDSCode/dependency_config.py

```diff
@@ -628,173 +638,84 @@
         primary_descriptor="45sensor-rates",
         dependent_instrument="ultra",
         dependent_data_level="l1b",
-        dependent_descriptor="45all",
-        relationship="HARD",
-        direction="DOWNSTREAM",
-    ),
-    # TODO: uncomment for post-SIT-3
-    # PreProcessingDependency(
-    #     primary_instrument="codice",
-    #     primary_data_level="l2",
-    #     primary_descriptor="todo",
-    #     dependent_instrument="ultra",
-    #     dependent_data_level="l1b",
-    #     dependent_descriptor="45all",
-    #     relationship="HARD",
-    #     direction="DOWNSTREAM",
-    # ),
-    # PreProcessingDependency(
-    #     primary_instrument="swapi",
-    #     primary_data_level="l2",
-    #     primary_descriptor="todo",
-    #     dependent_instrument="ultra",
-    #     dependent_data_level="l1b",
-    #     dependent_descriptor="45all",
-    #     relationship="HARD",
-    #     direction="DOWNSTREAM",
-    # ),
-    PreProcessingDependency(
-        primary_instrument="ultra",
-        primary_data_level="l1b",
-        primary_descriptor="45sensor-de",
-        dependent_instrument="ultra",
-        dependent_data_level="l1c",
-        dependent_descriptor="45all",
+        dependent_descriptor="45sensor-extendedspin",
         relationship="HARD",
         direction="DOWNSTREAM",
     ),
+    # TODO: ask difference of following two products
     PreProcessingDependency(
```

Comment:
The way it works now we only need l1a sensor rates to kick off l1b extended spin. Kind of like your question about pset. There isn't a reason to kickoff another batch job since cullingmask and badtimes are the exact same thing as extended spin, but with quality flags.

Reply:
So lines 645-665 can be deleted.

Reply:
cool. That makes sense. Will remove that!